### PR TITLE
feat: Add Linter Environment

### PR DESCRIPTION
**Pull request by dkimura**
## TL;DR

- Add ESLint and Prettier Environment

## Check list

- [x] Passed CI
- [x] Enable Linter on VSC

### CODE REVIEW

File: packages/eslint-config-globis/package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
   },
   "peerDependencies": {
     "eslint": "^6.7.2",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "typescript": "^3.7.4"
```
Comment (shoota) [inline packages/eslint-config-globis/package.json:30]: nice

File: packages/eslint-config-globis/base.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['airbnb/base', './shared.js'],
```
Comment (dkimura) [inline packages/eslint-config-globis/base.js:2]: made a rule ignores react and hooks.

**Review by shoota — APPROVED** [review]
LGTM

**Review by tatsushitoji — APPROVED** [review]
LGTM 👍

**Pull request merged**